DigitalLearningSolutions.Data.Tests/Services/CourseContentServiceTests.cs

```diff
@@ -420,7 +420,7 @@ public void Get_course_content_for_refreshed_course_should_return_new_course()
                     4169,
                     "Level 2 - Microsoft Excel 2010",
                     "MOS Excel 2010 CORE",
-                    "5h 49m",
+                    349,

```

Comment:
Could we add a few tests comparing the new value to the output of the old scalar function? E.g. a `TestCase` test taking customisation id, new value (int), old value (string) as parameters, if that makes sense? That would make me a bit more confident that we haven't broken anything 🙂

Reply:
I've added some tests in 1c07aee 🙂

Whilst doing this I noticed that the duration can be null (for a course without tutorials). The function in the database would render this as "N/A", so course duration would read "Average Course Duration: N/A". This is what I've currently put into the new version, is this the behaviour you would like? Alternatively we could remove that line if there is no duration for the course 🙂

Reply:
Good question. I think leave it for now and I'll check with Kevin when he's back 🙂